test/acceptance/wait_for_test.go

```diff
@@ -0,0 +1,54 @@
+package provider
+
+import (
+	"testing"
+	"time"
+
+	tfstatehelper "github.com/hashicorp/terraform-provider-kubernetes-alpha/test/helper/state"
+)
+
+func TestKubernetesManifest_WaitForFields(t *testing.T) {
```

Comment:
This test is going to be flaky until we implement waiting for resource deletion.

Reply:
Yeah, I observed some flakiness here. We might need to create a new github issue and prioritize the `wait` for resource deletion. In my recent experience, a flaky delete can cause a bad user experience when resources are replaced during apply.